qiskit/providers/aer/backends/aerbackend.py

```diff
@@ -186,6 +190,18 @@ def _validate(self, qobj, backend_options, noise_model):
         """Validate the qobj, backend_options, noise_model for the backend"""
         pass
 
+    def _save_job(self, result, backend_options):
+        if (backend_options is not None) and ("path_to_save" in backend_options):
+            path_to_save = backend_options["path_to_save"]
```

Comment:
Per last [comment](https://github.com/Qiskit/qiskit-aer/issues/240#issuecomment-694115558) , let's just use a hardcoded path directory. A good candidate could be: `$HOME/.qiskit/aer_jobs/`.

Reply:
I agree we shouldn't be taking this in as a backend option. But besides a hardcoded path it should override-able with an env variable. You can leverage the settings.conf as mentioned in the issue down the road but for something like this you don't not want a hardcoded path without giving any option to override it easily. For example on some of my systems home directory space is intentionally very limited and always writing to `/home` is not viable.

That being said we still also should make saving results an opt-in feature. It's not something everyone will want and does add overhead (both serialization and disk io).

Reply:
@mtreinish @atilag understood :+1: Let me look into leveraging `settings.conf`.
> That being said we still also should make saving results an opt-in feature

Yes, I plan on adding a `backend_option` called `save_job` which takes a boolean value indicating if that specific job needs to be saved. Does that make more sense?

Reply:
I see `terra` [has a class to read the config file](https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/user_config.py), but aer doesn't seem to have one at the moment. Should I add this as well? I can modify the [read_config_file](https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/user_config.py#L48) function to just read the directory to be used for saving and leave the terra stuff untouched.